src/Data/Functor/Plus.hs

```diff
@@ -97,12 +99,26 @@ class Alt f => Plus f where
 psum :: (Foldable t, Plus f) => t (f a) -> f a
 psum = foldr (<!>) zero
 
+-- | Generic 'zero'. Caveats:
+--
+--   1. Will not compile if @f@ is a sum type.
+--   2. Will not compile if @f@ is a recursive type.
```

Comment:
Luckily, I think we can give `(:.:)` a `Plus` instance by adapting the existing one for `Compose`:

https://github.com/ekmett/semigroupoids/blob/bcf6c0f2ec29c688417f316c270fdf9ea2df866d/src/Data/Functor/Plus.hs#L207-L208

This would allow us to delete this caveat.

Reply:
Done.